crypto/asn1/a_time.c

```diff
@@ -79,6 +84,12 @@ int asn1_time_to_tm(struct tm *tm, const ASN1_TIME *d)
     char *a;
     int n, i, i2, l, o, min_l = 11, strict = 0, end = 6, btz = 5, md;
     struct tm tmp;
+  const char
+    upper_z = 0x5A,
+    num_zero = 0x30,
+    period = 0x2E,
+    hyphen = 0x2D,
+    plus = 0x2B;
```

Comment:
Please put all of these on a single line (breaking at 80 chars)

Reply:
Done